src/components/tests/content-tab-assistant.test.tsx

```diff
@@ -42,6 +42,9 @@ describe( 'ContentTabAssistant', () => {
 	} );
 
 	const authenticate = jest.fn();
+
+	const getInput = () => screen.getByTestId( 'ai-input-textarea' ) as HTMLInputElement;
```

Comment:
Are we able to use a user-facing query instead? E.g., the placeholder or ARIA label. Doing so is a more stable and user-oriented approach IMO.